www/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
```

Comment:
We should move this to the frontend that Brady has set up so that we can view it inside of our existing UI.

Reply:
I am happy to work with brady to integrate this to UI, I was unsure whether he wanted to work on the websocket part of the code. This webpage would need to be served somewhere in the frontend project and need the same event handlers and DOM elements. 

If it becomes a long term project I suggest leaving this folder as is because this pages are just html and vanilla JS and there are not a lot of packages here subject to change which would affect future commits.